bpf/lib/nodeport.h

```diff
@@ -1116,11 +1117,11 @@ static __always_inline bool snat_v4_needed(struct __ctx_buff *ctx, __be32 *addr,
 	 * cluster.
 	 */
 	if (1) {
```

Comment:
While we are here - any reason for `if (1)`?

Reply:
I don't think so, seems to me it's just a way of defining a scope.